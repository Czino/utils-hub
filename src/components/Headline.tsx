export const Headline = ({ className, children }: React.ComponentProps<'h1'>) => (
  <h1 className={['text-5xl', className].join(' ')}>{children}</h1>
)
