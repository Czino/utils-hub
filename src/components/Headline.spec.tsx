import { render } from '@testing-library/react'
import { Headline } from './Headline'

describe('Headline', () => {
  it('should render Headline with styles', () => {
    const { asFragment } = render(<Headline className="mt-4">Test</Headline>)
    expect(asFragment()).toMatchSnapshot()
  })
})
