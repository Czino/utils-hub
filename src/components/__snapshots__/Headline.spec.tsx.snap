// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline should render Headline with styles 1`] = `
<DocumentFragment>
  <h1
    class="text-5xl mt-4"
  >
    Test
  </h1>
</DocumentFragment>
`;
